modules/gdscript/tests/gdscript_test_runner.cpp

```diff
@@ -251,7 +251,10 @@ bool GDScriptTestRunner::make_tests_for_dir(const String &p_dir) {
 				return false;
 			}
 		} else {
-			if (next.get_extension().to_lower() == "gd") {
+			if (next.ends_with(".notest.gd")) {
```

Comment:
I added the `.notest.gd` exception here, as it was defined elsewhere, but not used. Practical for `.gd` files that are loaded by tests, but not tests themselves.